samples/dotnet/samples/pwsh/Test/ResourceGroup.Tests.ps1

```diff
@@ -0,0 +1,42 @@
+$ROOT_PATH = $PSScriptRoot | split-path -parent | split-path -parent | split-path -parent | split-path -parent | split-path -parent
```

Comment:
Why 5 occurences of `split-path -parent` ? It seems like it would be for relative pathing?

Reply:
$PSScriptRoot is relative to where the program is executed. since my test file is located in samples/dotnet/samples/pwsh/Test, that is the one way I am referencing to the repo root path.